@navikt/core/react/src/form/combobox/combobox.stories.tsx

```diff
@@ -35,7 +35,11 @@ const options = [
 ];
 
 export const Default: StoryFunction = (props) => (
-  <UNSAFE_Combobox {...props} id="combobox" />
+  <UNSAFE_Combobox
+    {...props}
+    maxSelected={{ limit: Number(props.maxSelected) }}
```

Comment:
Hvorfor maxSelected på default-eksempelet?

Reply:
Tried to test edge cases, like having that prop active in Storybook. But it didn't work from the Storybook-controls, so that was the fix 👍